framework/include/security/security_common.h

```diff
@@ -200,6 +200,10 @@ typedef struct _security_ecdh_param {
  */
 security_error security_init(security_handle *hnd);
 security_error security_deinit(security_handle hnd);
+#ifdef CONFIG_ARM_TRUSTZONE
```

Comment:
if you want to add new APIs to framework then give us the document
the document have to contain the purpose of API, description to input parameters and return value.
Then we'll decide whether accept it or not.
Furthermore we don't allow to add a dependency like CONFIG_ARM_TRUSTZONE in common framework header files.

Reply:
As discussed earlier with @sunghan-chang , we needed to integrate RTOS related TZ APIs with SE. The PR is created based on that design agreed. I will create the document about API details